_projects/SP19-CulturalHeritageSites.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: project
+title: Cultural Heritage Sites (Cahokia Mounds)
+image:
+desc: Virtual experiene of Cahokia Mounds, with the environment created using LiDAR data.
+github:
+dl: https://drive.google.com/drive/folders/1dZ67RJ7ryCFAwCrcGxavylECdB9qpkZ2?usp=sharing
```

Comment:
The drive folder is missing a Data folder, needed to run the project

Reply:
Added Data folder and other stuff just in case